packages/components/src/font-size-picker/index.tsx

```diff
@@ -230,7 +230,11 @@ const UnforwardedFontSizePicker = (
 									onChange?.(
 										hasUnits
 											? selectedItem.size
-											: Number( selectedItem.size )
+											: Number( selectedItem.size ),
+										getSelectedOption(
```

Comment:
Would there be a better way to get the preset slug back to the callback?

The reason we need it is so we can build the `var:preset|font-size|${slug}` value.

Reply:
I quite like this approach in that you're passing the whole selected preset in the `onChange` callback. That means that the consumer can potentially do other things with that metadata, rather than passing only the slug.